lib/inets/test/inets_test_lib.erl

```diff
@@ -472,13 +469,10 @@ connect_byte(SockType, Host, Port) ->
     connect_byte(SockType, Host, Port, []).
     
 connect_byte(ssl, Host, Port, Opts0) ->
-    Opts = [{packet,0} | Opts0], 
-    connect(ssl, Host, Port, Opts);
-connect_byte(essl, Host, Port, Opts0) ->
-    Opts = [{ssl_imp, new}, {packet,0} | Opts0], 
+    Opts = [list, {packet,0} | Opts0], 
```

Comment:
is `list` option in `Opts` equivalent to `{mode, list}`?

Reply:
Yes, see gen_tcp and inets man page!